apps/passport-client/pages/index.tsx

```diff
@@ -424,7 +428,7 @@ async function loadInitialState(): Promise<AppState> {
 
   const persistentSyncStatus = loadPersistentSyncStatus();
 
-  return {
+  const state: AppState = {
```

Comment:
Not vital in this PR, but I'd really love to move the identity creation later so we don't always generate a semaphore key before the user logs in.  It seems like asking for a bug where the user gets the wrong identity.

Reply:
That makes sense to me.